CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+First off, thanks for taking the time to contribute! We welcome contributions
+from everyone, and this guide will answer some common questions about how this
+project works.
+
+## Making Changes
+
+1. Fork this repository to your own account
+2. Make your changes and verify that `mix test` passes
+3. Commit your work and push to a topic branch on your fork
+4. Submit a [pull request](https://github.com/pinterest/elixir-thrift/compare/)
```

Comment:
Should we mention what to expect once a PR is open? I think this is the most likely stage where expectations can diverge. Something like:
5. Participate in mutual review process hopefully leading to merging the contribution.